sw/airborne/firmwares/fixedwing/main_ap.c

```diff
@@ -108,6 +108,11 @@ PRINT_CONFIG_MSG_VALUE("USE_BARO_BOARD is TRUE, reading onboard baro: ", BARO_BO
 #define COMMAND_YAW_TRIM 0
 #endif
 
+/* Datalink lost time exception [s] */
+#ifndef DATALINK_LOST_TIME
+#define DATALINK_LOST_TIME 65534 // 45 days - i.e. no exception
```

Comment:
I don't like these default values.. Not that we ever reach this number, but it would be better to just add the `#ifndef` around the `if` below.